src/services/rfqm_service.ts

```diff
@@ -932,27 +940,33 @@ export class RfqmService {
 
             switch (jobStatus) {
                 case RfqmJobStatus.PendingSubmitted:
-                    const newGasPrice = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
+                    const gasPriceEstimate = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
```

Comment:
I'm pretty confused by this and by the overall strategy we're using here. Isn't this returning the Type 0 "fast gas" (i.e. the Base Fee + a buffer)?

I'm thinking we should use a strategy similar to the one outlined [here](https://www.blocknative.com/blog/eip-1559-fees): Submit the tx with a `maxFeePerGas` equal to 1 or 1.5 gwei `maxPriortyFeePerGas` + 2 * the base fee, then increase the `maxPriorityFeePerGas` if necessary. Alternatively, I thought we had a new library which can give us real-time estimates for the gas price.

Reply:
Ah, this is another thing that Mason and I discussed. I think this strategy will be "no worse" than what we currently have, with the upside that it caps our payment to the fast gas price

The issue if we give a large `maxFeePerGas` is that it will result in fairly different submission logic (i.e. on EIP-1559 chains, we can submit and wait and only update the `maxPriorityFeePerGas`, but other chains we do the replacing strategy as before)

If I'm not mistaken, I think this strategy allows us to have very similar submission logic for both types of chains